src/Microsoft.Management.Deployment/PackageManager.cpp

```diff
@@ -401,6 +401,11 @@ namespace winrt::Microsoft::Management::Deployment::implementation
             }
 
             // Note: AdditionalPackageCatalogArguments is not needed during install since the manifest is already known so no additional calls to the source are needed. The property is deprecated.
+
+            if (options.AcceptPackageAgreements())

```

Comment:
If InstallOptions is null, we are breaking previous behavior

Reply:
Modified this so that if installoptions is null we assume the package agreements are accepted as to not break previous behavior